ui/mirage/config.js

```diff
@@ -731,10 +731,6 @@ export default function () {
     });
   });
 
-  this.get('/namespaces', function ({ namespaces }, req) {
```

Comment:
This was overriding an existing call which had some custom logic that handled default ns. That is why those tests were failing I think.

Reply:
Damn... this wasn't it